migrations/20181011143503_create_instructions_table.js

```diff
@@ -0,0 +1,15 @@
+exports.up = function(knex, Promise) {
+  return knex.schema.createTable("instructions", table => {
+    table.increments();
+    table.string("steps");
```

Comment:
You'll want to create a separate `steps` table, and use this `instructions` table as a many-to-many table connecting steps to recipes.